### DISCUSSION TITLE
Antena 3

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#726] [Show and tell]
**Channel Name:** Antena 3
**Channel ID:** Antena3.es
**Country:** Spain
**Language:** Spanish
**Website:** https://www.antena3.com/
**Stream URL (optional):** http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/3c4f6df40f586ddb061fca1ff0ab0bf87973b442_1635468712/antena3/bitrate_4.m3u8
**Notes (optional):** It's an official Atresmedia stream. It has a token, but I think it has an automatic refresher for the https://photocall.tv page which is where I got it from. In this repository they do magic, so maybe they can make the token renew automatically. There are also other Atresmedia links, such as LaSexta, Neox, Nova, Mega & A3Series.
If you allow me and I can attach them, here they go:
LaSexta: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/70f935154f65972e9f244c54048105a07ea2409c_1635457355/lasexta/bitrate_4.m3u8
Neox: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/518d9463e7a2c3edfd16a2bcb72540814d3deec7_1635463091/geoneox/bitrate_4.m3u8
Nova: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/7908849782ea2fb912be8b7dc4fa25b8158150ca_1635463113/geonova/bitrate_4.m3u8
Mega: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/69271977a08cc6e26dd4b72ad7de47e406b48a38_1635453173/geomega/bitrate_4.m3u8
A3Series: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/7213eb9f0f6c91dea8cff0c3565828553e69e170_1635453132/geoa3series/bitrate_4.m3u8

**Comment by BellezaEmporium** [comment]
Adaptives : 

A3 : http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/3c4f6df40f586ddb061fca1ff0ab0bf87973b442_1635468712/antena3/master.m3u8

LaSexta : http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/70f935154f65972e9f244c54048105a07ea2409c_1635457355/lasexta/master.m3u8

Neox: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/518d9463e7a2c3edfd16a2bcb72540814d3deec7_1635463091/geoneox/master.m3u8

Mega: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/69271977a08cc6e26dd4b72ad7de47e406b48a38_1635453173/geomega/master.m3u8

A3Series: http://hls.webredirect.org/h4gs/mp_4953.php/https://directo.atresmedia.com/7213eb9f0f6c91dea8cff0c3565828553e69e170_1635453132/geoa3series/master.m3u8

**Comment by robocopz** [comment]
Proxy and token changing = useless links

**Comment by BellezaEmporium** [comment]
> 
> 
> Proxy and token changing = useless links

Proxy isn't an issue here. Token changing however is. Might check how could I get a fresh link every time.

**Comment by BellezaEmporium** [comment]
I have different links don't ask me why x)

Antena 3 is https://channel-antena3-cdn.blim.com/manifest.mpd / https://channel-antena3-cdn.blim.com/manifest.m3u8

**Comment by BellezaEmporium** [comment]
I might get some lives from Mitele ES, however it's going to cost lots of memory power, it scrapes a link for certain tokens, scrapes a link for a certain "cerbero" URL that changes over time, then sends a POST to it with both tokens to get the Akamai tokens, then you add it next to the Mitele link you're watching, and you're good to go.

Aaaaah, Mediaset...

For example : Telecinco.

1) Mediaset server GET : https://caronte.mediaset.es/delivery/channel/mmc/telecinco/mtweb

Results = cerbero link + bbx token needed.

2) Other Mediaset server GET : https://mab.mediaset.es/1.0.0/get?oid=mtmw&eid=%2Fapi%2Fmtmw%2Fv2%2Fgbx%2Fmtweb%2Flive%2Fmmc%2Ftelecinco

Results = gbx token

3) POST on Cerbero links with bbx and gbx token

Results = our Akamai data

4) paste it next to your Mitele live link (you can also get it from the 1st request, but that doesn't change)

5) enjoy

**Comment by GiovanniSebastian** [comment]
Blim links don't work for me

**Comment by robocopz** [comment]
> I might get some lives from Mitele ES, however it's going to cost lots of memory power, it scrapes a link for certain tokens, scrapes a link for a certain "cerbero" URL that changes over time, then sends a POST to it with both tokens to get the Akamai tokens, then you add it next to the Mitele link you're watching, and you're good to go.
> 
> Aaaaah, Mediaset...
> 
> For example : Telecinco.
> 
> 1. Mediaset server GET : https://caronte.mediaset.es/delivery/channel/mmc/telecinco/mtweb
> 
> Results = cerbero link + bbx token needed.
> 
> 1. Other Mediaset server GET : https://mab.mediaset.es/1.0.0/get?oid=mtmw&eid=%2Fapi%2Fmtmw%2Fv2%2Fgbx%2Fmtweb%2Flive%2Fmmc%2Ftelecinco
> 
> Results = gbx token
> 
> 1. POST on Cerbero links with bbx and gbx token
> 
> Results = our Akamai data
> 
> 1. paste it next to your Mitele live link (you can also get it from the 1st request, but that doesn't change)
> 2. enjoy

Good for local but not for remote. You can't bypass the ~ ip = in the m3u8 now.

**Comment by BellezaEmporium** [comment]
> 
> 
> > I might get some lives from Mitele ES, however it's going to cost lots of memory power, it scrapes a link for certain tokens, scrapes a link for a certain "cerbero" URL that changes over time, then sends a POST to it with both tokens to get the Akamai tokens, then you add it next to the Mitele link you're watching, and you're good to go.
> > Aaaaah, Mediaset...
> > For example : Telecinco.
> > 
> > 1. Mediaset server GET : https://caronte.mediaset.es/delivery/channel/mmc/telecinco/mtweb
> > 
> > Results = cerbero link + bbx token needed.
> > 
> > 1. Other Mediaset server GET : https://mab.mediaset.es/1.0.0/get?oid=mtmw&eid=%2Fapi%2Fmtmw%2Fv2%2Fgbx%2Fmtweb%2Flive%2Fmmc%2Ftelecinco
> > 
> > Results = gbx token
> > 
> > 1. POST on Cerbero links with bbx and gbx token
> > 
> > Results = our Akamai data
> > 
> > 1. paste it next to your Mitele live link (you can also get it from the 1st request, but that doesn't change)
> > 2. enjoy
> 
> Good for local but not for remote. You can't bypass the ~ ip = in the m3u8 now.

Yep, unless I put a proxy on both sides.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/es.m3u#L51-L52
